diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/Wnn/man/2.env/dic/Makefile.in FreeWnn-1.1.1-a023/Wnn/man/2.env/dic/Makefile.in
--- FreeWnn-1.1.1-a023.org/Wnn/man/2.env/dic/Makefile.in	2001-08-14 15:43:21.000000000 +0200
+++ FreeWnn-1.1.1-a023/Wnn/man/2.env/dic/Makefile.in	2023-02-02 20:01:12.676502036 +0100
@@ -34,6 +34,7 @@ top_builddir = ../../../..
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 
+MANDIR = $(JMANDIR)
 MANSUFFIX = 5
 
 all::
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/Wnn/man/2.env/wnn/Makefile.in FreeWnn-1.1.1-a023/Wnn/man/2.env/wnn/Makefile.in
--- FreeWnn-1.1.1-a023.org/Wnn/man/2.env/wnn/Makefile.in	2001-08-14 15:43:21.000000000 +0200
+++ FreeWnn-1.1.1-a023/Wnn/man/2.env/wnn/Makefile.in	2023-02-02 20:01:12.676502036 +0100
@@ -34,6 +34,7 @@ top_builddir = ../../../..
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 
+MANDIR = $(JMANDIR)
 MANSUFFIX = 4
 
 install.man:: 2a_ctrl.man
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/Wnn/man/3.libwnn/jl_lib/Makefile.in FreeWnn-1.1.1-a023/Wnn/man/3.libwnn/jl_lib/Makefile.in
--- FreeWnn-1.1.1-a023.org/Wnn/man/3.libwnn/jl_lib/Makefile.in	2001-09-16 16:20:17.000000000 +0200
+++ FreeWnn-1.1.1-a023/Wnn/man/3.libwnn/jl_lib/Makefile.in	2023-02-02 20:01:12.676502036 +0100
@@ -34,7 +34,7 @@ top_builddir = ../../../..
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 
-MANDIR = $(LIBMANDIR)
+MANDIR = $(JLIBMANDIR)
 MANSUFFIX = $(LIBMANSUFFIX)
 
 install.man:: bun_suu.man
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/Wnn/man/3.libwnn/js_lib/Makefile.in FreeWnn-1.1.1-a023/Wnn/man/3.libwnn/js_lib/Makefile.in
--- FreeWnn-1.1.1-a023.org/Wnn/man/3.libwnn/js_lib/Makefile.in	2001-08-14 15:43:21.000000000 +0200
+++ FreeWnn-1.1.1-a023/Wnn/man/3.libwnn/js_lib/Makefile.in	2023-02-02 20:01:12.676502036 +0100
@@ -34,7 +34,7 @@ top_builddir = ../../../..
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 
-MANDIR = $(LIBMANDIR)
+MANDIR = $(JLIBMANDIR)
 MANSUFFIX = $(LIBMANSUFFIX)
 
 all::
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/Wnn/man/3.libwnn/misc_lib/Makefile.in FreeWnn-1.1.1-a023/Wnn/man/3.libwnn/misc_lib/Makefile.in
--- FreeWnn-1.1.1-a023.org/Wnn/man/3.libwnn/misc_lib/Makefile.in	2001-08-14 15:43:21.000000000 +0200
+++ FreeWnn-1.1.1-a023/Wnn/man/3.libwnn/misc_lib/Makefile.in	2023-02-02 20:01:12.676502036 +0100
@@ -34,7 +34,7 @@ top_builddir = ../../../..
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 
-MANDIR = $(LIBMANDIR)
+MANDIR = $(JLIBMANDIR)
 MANSUFFIX = $(LIBMANSUFFIX)
 
 install.man:: msg_open.man
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/Wnn/man/3.libwnn/rk_lib/Makefile.in FreeWnn-1.1.1-a023/Wnn/man/3.libwnn/rk_lib/Makefile.in
--- FreeWnn-1.1.1-a023.org/Wnn/man/3.libwnn/rk_lib/Makefile.in	2001-08-14 15:43:21.000000000 +0200
+++ FreeWnn-1.1.1-a023/Wnn/man/3.libwnn/rk_lib/Makefile.in	2023-02-02 20:01:12.676502036 +0100
@@ -34,7 +34,7 @@ top_builddir = ../../../..
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 
-MANDIR = $(LIBMANDIR)
+MANDIR = $(JLIBMANDIR)
 MANSUFFIX = $(LIBMANSUFFIX)
 
 install.man:: rk_clear.man
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/Wnn/man/4.cmd/Makefile.in FreeWnn-1.1.1-a023/Wnn/man/4.cmd/Makefile.in
--- FreeWnn-1.1.1-a023.org/Wnn/man/4.cmd/Makefile.in	2010-07-11 12:18:17.000000000 +0200
+++ FreeWnn-1.1.1-a023/Wnn/man/4.cmd/Makefile.in	2023-02-02 20:01:12.676502036 +0100
@@ -35,6 +35,7 @@ top_builddir = @top_builddir@
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 
+MANDIR = $(JMANDIR)
 MANSUFFIX = 1
 
 SERVER_MAN = jserver.man
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/Wnn/man/6.jutil/Makefile.in FreeWnn-1.1.1-a023/Wnn/man/6.jutil/Makefile.in
--- FreeWnn-1.1.1-a023.org/Wnn/man/6.jutil/Makefile.in	2010-07-11 12:18:17.000000000 +0200
+++ FreeWnn-1.1.1-a023/Wnn/man/6.jutil/Makefile.in	2023-02-02 20:01:12.676502036 +0100
@@ -35,6 +35,7 @@ top_builddir = @top_builddir@
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 
+MANDIR = $(JMANDIR)
 MANSUFFIX = 1
 
 MANUALS = atod.man dtoa.man atof.man wddel.man wdreg.man wnntouch.man
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/makerule.mk.in FreeWnn-1.1.1-a023/makerule.mk.in
--- FreeWnn-1.1.1-a023.org/makerule.mk.in	2023-02-02 20:01:12.163168702 +0100
+++ FreeWnn-1.1.1-a023/makerule.mk.in	2023-02-02 20:01:12.676502036 +0100
@@ -48,6 +48,8 @@ all::
 
            MANDIR = @mandir@/man$(MANSUFFIX)
         LIBMANDIR = @mandir@/man$(LIBMANSUFFIX)
+          JMANDIR = @mandir@/ja/man$(MANSUFFIX)
+       JLIBMANDIR = @mandir@/ja/man$(LIBMANSUFFIX)
 
         MANSUFFIX = 1
      LIBMANSUFFIX = 3
