--- FreeWnn-1.1.1-a013.orig/makerule.mk.in	Wed Sep 29 01:56:24 1999
+++ FreeWnn-1.1.1-a013/makerule.mk.in	Mon Jan 31 13:26:50 2000
@@ -113,7 +113,7 @@
          prefix = @prefix@
     exec_prefix = @exec_prefix@
          PREFIX = @prefix@
-      WNNWNNDIR = ${PREFIX}/lib/wnn
+      WNNWNNDIR = /etc/FreeWnn
       WNNBINDIR = ${PREFIX}/bin
       WNNINCDIR = @includedir@
       WNNLIBDIR = @libdir@
@@ -125,9 +125,9 @@
          WNNTOP = $(IMTOP)/Wnn
-       JWNNLANG = ja_JP
+       JWNNLANG = ja
      JWNNWNNDIR = $(WNNWNNDIR)/$(JWNNLANG)
-     JWNNBINDIR = $(WNNBINDIR)/Wnn4
+     JWNNBINDIR = $(WNNBINDIR)
      JWNNINCDIR = $(WNNINCDIR)/wnn
-     JWNNDICDIR = $(JWNNWNNDIR)/dic
+     JWNNDICDIR = /var/lib/wnn/$(JWNNLANG)/dic
   JWNNPUBDICDIR = $(JWNNDICDIR)/pubdic
 JWNNWNNCONSDICDIR = $(JWNNDICDIR)/wnncons
   JWNNFZKSRCDIR = $(JWNNDICDIR)/src
@@ -155,8 +155,8 @@
        CWNNLANG = zh_CN
      CWNNWNNDIR = $(WNNWNNDIR)/$(CWNNLANG)
      CWNNINCDIR = $(WNNINCDIR)/cwnn
-     CWNNBINDIR = $(WNNBINDIR)/cWnn4
-     CWNNDICDIR = $(CWNNWNNDIR)/dic
+     CWNNBINDIR = $(WNNBINDIR)
+     CWNNDICDIR = /var/lib/wnn/$(CWNNLANG)/dic
   CWNNSYSDICDIR = $(CWNNDICDIR)/sys
       CWNNCDSRC = $(CWNNTOP)/cd
     CWNNCDICSRC = $(CWNNTOP)/cdic
@@ -178,8 +178,8 @@
        KWNNLANG = ko_KR
      KWNNWNNDIR = $(WNNWNNDIR)/$(KWNNLANG)
      KWNNINCDIR = $(WNNINCDIR)/kwnn
-     KWNNBINDIR = $(WNNBINDIR)/kWnn4
-     KWNNDICDIR = $(KWNNWNNDIR)/dic
+     KWNNBINDIR = $(WNNBINDIR)
+     KWNNDICDIR = /var/lib/wnn/$(KWNNLANG)/dic
   KWNNSYSDICDIR = $(KWNNDICDIR)/sys
       KWNNKDSRC = $(KWNNTOP)/kd
     KWNNKDICSRC = $(KWNNTOP)/kdic
@@ -199,7 +199,7 @@
 
        TWNNLANG = zh_TW
      TWNNWNNDIR = $(WNNWNNDIR)/$(TWNNLANG)
-     TWNNDICDIR = $(TWNNWNNDIR)/dic
+     TWNNDICDIR = /var/lib/wnn/$(TWNNLANG)/dic
   TWNNSYSDICDIR = $(TWNNDICDIR)/sys
       TWNNTDSRC = $(CWNNTOP)/td
     TWNNTDICSRC = $(CWNNTOP)/tdic
