--- FreeWnn-1.1.1-a021/Wnn/jserver/de.c.orig	2012-11-16 20:41:53.701176783 +0100
+++ FreeWnn-1.1.1-a021/Wnn/jserver/de.c	2012-11-16 20:45:47.327838583 +0100
@@ -240,7 +240,8 @@
   /* option default */
   option_flag = (OPT_FORK & ~OPT_VERBOSE);
 
-  setuid (geteuid ());
+  setregid(getegid(), getegid());
+  setreuid(geteuid(), geteuid());
   get_options (argc, argv);
   print_version();
   log_debug("invoked as %s.", argv[0]);
--- FreeWnn-1.1.1-a023/Wnn/uum/jhlp.c.orig	2016-09-27 18:18:54.713645559 +0200
+++ FreeWnn-1.1.1-a023/Wnn/uum/jhlp.c	2016-09-27 18:37:20.170299605 +0200
@@ -1661,8 +1661,8 @@
       ioctl (ttyfd, TIOCSWINSZ, &win);
 #endif
 #ifdef linux
-      setgid (getgid ());
-      setuid (getuid ());
+      setregid (getegid (), getegid ());
+      setreuid (geteuid (), geteuid ());
 #endif
       execvp (cmdnm, argv);
       uum_err ("exec fault.");
