--- FreeWnn-1.1.1-a021/Contrib/dic/gerodic/Makefile.in.orig	2012-11-16 19:46:06.097913226 +0100
+++ FreeWnn-1.1.1-a021/Contrib/dic/gerodic/Makefile.in	2012-11-16 19:49:16.601242597 +0100
@@ -43,7 +43,7 @@
 
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
-LOCAL_INSTFLAGS = -o ${WNNOWNER}
+LOCAL_INSTFLAGS =
 
 #ATOD=${JWNNBINDIR}/atod
 ATOD=${WNNJUTILSRC}/atod
@@ -67,7 +67,6 @@
 install:: ${OBJ}
 	@if [ -d $(DESTDIR)$(GERODIR) ]; then set +x; \
 	else (set -x; $(MKDIRHIER) $(DESTDIR)$(GERODIR)); fi
-	chown ${WNNOWNER} $(DESTDIR)$(GERODIR)
 	$(INSTALL) $(INSTDATFLAGS) $(LOCAL_INSTFLAGS) ${OBJ} $(DESTDIR)$(GERODIR)
 	${WNNTOUCH} $(DESTDIR)$(GERODIR)/${OBJ}
 
--- FreeWnn-1.1.1-a021/Wnn/jserver/Makefile.in.orig	2012-11-16 19:46:06.101246559 +0100
+++ FreeWnn-1.1.1-a021/Wnn/jserver/Makefile.in	2012-11-16 19:49:41.477908774 +0100
@@ -36,7 +36,7 @@
 top_builddir = @top_builddir@
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
 
  JSERVERDEFINES = -DJSERVER -DWRITE_CHECK
        INCLUDES = -I$(WNNINCLUDESRC) -I$(top_builddir) -I$(srcdir)
--- FreeWnn-1.1.1-a021/Wnn/jutil/Makefile.in.orig	2012-11-16 19:46:06.101246559 +0100
+++ FreeWnn-1.1.1-a021/Wnn/jutil/Makefile.in	2012-11-16 19:50:18.187907603 +0100
@@ -37,7 +37,7 @@
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 LT_INSTALL      = $(LIBTOOL) --mode=install $(INSTALL)
 LT_LD           = $(LIBTOOL) --mode=link $(CCLINK)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
 
        INCLUDES = -I$(WNNINCLUDESRC) -I$(top_builddir)
         DEFINES = $(WNNDEFINES) $(WNNLANGDEF)
--- FreeWnn-1.1.1-a021/Wnn/pubdicplus/Makefile.in.orig	2004-04-24 20:56:40.000000000 +0200
+++ FreeWnn-1.1.1-a021/Wnn/pubdicplus/Makefile.in	2012-11-16 19:50:33.974574401 +0100
@@ -35,7 +35,7 @@
  WNNJUTILOBJ = $(top_builddir)/Wnn/jutil
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
 
        INCLUDES = -I$(top_builddir)
            ATOD = $(WNNJUTILOBJ)/atod
@@ -377,8 +377,7 @@
 install::
 	@case '${MFLAGS}' in *[i]*) set +e;; esac;
 	@for i in $(JWNNDICDIR); do if [ -d $(DESTDIR)$$i ]; then \
-	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i; \
-	chown $(WNNOWNER) $(DESTDIR)$$i); fi \
+	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i); fi \
 	done
 
 install:: $(DIC)
--- FreeWnn-1.1.1-a021/Wnn/wnncons/dic/Makefile.in.orig	2001-08-14 15:43:21.000000000 +0200
+++ FreeWnn-1.1.1-a021/Wnn/wnncons/dic/Makefile.in	2012-11-16 19:51:15.051240552 +0100
@@ -33,7 +33,7 @@
 top_builddir = ../../..
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
 
            ATOD = $(WNNJUTILSRC)/atod
        WNNTOUCH = $(WNNJUTILSRC)/wnntouch
@@ -74,8 +74,7 @@
 install::
 	@case '${MFLAGS}' in *[i]*) set +e;; esac;
 	@for i in $(JWNNDICDIR); do if [ -d $(DESTDIR)$$i ]; then \
-	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i; \
-	chown $(WNNOWNER) $(DESTDIR)$$i); fi \
+	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i); fi \
 	done
 
 install:: $(DICS)
--- FreeWnn-1.1.1-a021/Xwnmo/xjutil/Makefile.in.orig	2012-11-16 19:46:06.107913226 +0100
+++ FreeWnn-1.1.1-a021/Xwnmo/xjutil/Makefile.in	2012-11-16 19:52:15.237905547 +0100
@@ -26,7 +26,7 @@
 top_builddir = ../..
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
 
        INCLUDES = -I. $(XWNMOINCLUDES) -I$(XWNMOSRC)
 
--- FreeWnn-1.1.1-a021/Xwnmo/xwnmo/Makefile.in.orig	2012-11-16 19:46:06.111246559 +0100
+++ FreeWnn-1.1.1-a021/Xwnmo/xwnmo/Makefile.in	2012-11-16 19:52:35.257905017 +0100
@@ -26,7 +26,7 @@
 top_builddir = ../..
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
 
        INCLUDES = $(XWNMOINCLUDES)
 
--- FreeWnn-1.1.1-a021/cWnn/cdic/Makefile.in.orig	2004-04-24 20:56:40.000000000 +0200
+++ FreeWnn-1.1.1-a021/cWnn/cdic/Makefile.in	2012-11-16 19:53:08.247904391 +0100
@@ -31,7 +31,7 @@
 CWNNJUTILOBJ = $(top_builddir)/cWnn/jutil
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
 
         ATOD = $(CWNNJUTILOBJ)/catod
         ATOF = $(CWNNJUTILOBJ)/catof
@@ -85,8 +85,7 @@
 install::
 	@case '${MFLAGS}' in *[i]*) set +e;; esac;
 	@for i in $(CWNNDICDIR); do if [ -d $(DESTDIR)$$i ]; then \
-	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i; \
-	chown $(WNNOWNER) $(DESTDIR)$$i); fi \
+	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i); fi \
 	done
 
 install:: $(ALLTGT)
--- FreeWnn-1.1.1-a021/cWnn/jserver/Makefile.in.orig	2004-05-21 18:39:32.000000000 +0200
+++ FreeWnn-1.1.1-a021/cWnn/jserver/Makefile.in	2012-11-16 19:46:06.114579892 +0100
@@ -38,7 +38,7 @@
  JSERVERDEFINES = -DJSERVER -DWRITE_CHECK
        INCLUDES = -I$(CWNNINCLUDESRC) -I$(top_builddir)
         DEFINES = $(CWNNDEFINES) $(JSERVERDEFINES) $(WNNTERM) $(CWNNLANGDEF)
-LOCAL_INSTFLAGS = $(INSTUIDFLAGS) -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
   SERVERINITDEF = -DSERVER_INIT_FILE=\"/$(CWNNLANG)/cserverrc\"
     HINSIDEFINE = -DHINSIDATA_FILE=\"/$(CWNNLANG)/cixing.data\"
  TSERVERINITDEF = -DSERVER_INIT_FILE=\"/$(TWNNLANG)/tserverrc\"
--- FreeWnn-1.1.1-a021/cWnn/jutil/Makefile.in.orig	2012-11-16 19:46:06.114579892 +0100
+++ FreeWnn-1.1.1-a021/cWnn/jutil/Makefile.in	2012-11-16 19:53:19.394570865 +0100
@@ -36,7 +36,7 @@
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 LT_INSTALL      = $(LIBTOOL) --mode=install $(INSTALL)
 LT_LD           = $(LIBTOOL) --mode=link $(CCLINK)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
 
        INCLUDES = -I$(CWNNINCLUDESRC) -I$(top_builddir)
         DEFINES = $(CWNNDEFINES) $(CWNNLANGDEF)
--- FreeWnn-1.1.1-a021/cWnn/tdic/Makefile.in.orig	2004-04-24 20:56:40.000000000 +0200
+++ FreeWnn-1.1.1-a021/cWnn/tdic/Makefile.in	2012-11-16 19:53:49.317903600 +0100
@@ -31,7 +31,7 @@
 CWNNJUTILOBJ = $(top_builddir)/cWnn/jutil
 
 INSTALL      = @INSTALL@
-INSTALLFLAGS = -o $(WNNOWNER)
+INSTALLFLAGS =
 
         ATOD = $(CWNNJUTILOBJ)/catod
         ATOF = $(CWNNJUTILOBJ)/catof
@@ -85,8 +85,7 @@
 install::
 	@case '${MFLAGS}' in *[i]*) set +e;; esac;
 	@for i in $(TWNNDICDIR); do if [ -d $(DESTDIR)$$i ]; then \
-	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i; \
-	chown $(WNNOWNER) $(DESTDIR)$$i); fi \
+	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i); fi \
 	done
 
 install:: $(ALLTGT)
--- FreeWnn-1.1.1-a021/kWnn/jserver/Makefile.in.orig	2004-05-21 18:39:32.000000000 +0200
+++ FreeWnn-1.1.1-a021/kWnn/jserver/Makefile.in	2012-11-16 19:46:06.121246559 +0100
@@ -38,7 +38,7 @@
  JSERVERDEFINES = -DJSERVER -DWRITE_CHECK
        INCLUDES = -I$(KWNNJSERVERSRC) -I$(KWNNINCLUDESRC) -I$(top_builddir)
         DEFINES = $(KWNNDEFINES) $(JSERVERDEFINES) $(WNNTERM) $(KWNNLANGDEF)
-LOCAL_INSTFLAGS = $(INSTUIDFLAGS) -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
   SERVERINITDEF = -DSERVER_INIT_FILE=\"/$(KWNNLANG)/kserverrc\"
     HINSIDEFINE = -DHINSIDATA_FILE=\"/$(KWNNLANG)/hinsi.data\"
         WRAPLIB = @WRAPLIB@
--- FreeWnn-1.1.1-a021/kWnn/jutil/Makefile.in.orig	2012-11-16 19:46:06.121246559 +0100
+++ FreeWnn-1.1.1-a021/kWnn/jutil/Makefile.in	2012-11-16 19:54:51.017902278 +0100
@@ -39,7 +39,7 @@
 
        INCLUDES = -I$(KWNNINCLUDESRC) -I$(top_builddir)
         DEFINES = $(KWNNDEFINES) $(KWNNLANGDEF)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
     HINSIDEFINE = -DHINSIDATA_FILE=\"/$(KWNNLANG)/hinsi.data\"
 
 CLIENTPROGRAMS = kwnnstat kwnnkill kwddel kwdreg
--- FreeWnn-1.1.1-a021/kWnn/kdic/Makefile.in.orig	2004-04-24 20:56:40.000000000 +0200
+++ FreeWnn-1.1.1-a021/kWnn/kdic/Makefile.in	2012-11-16 19:55:04.417902006 +0100
@@ -31,7 +31,7 @@
 KWNNJUTILOBJ = $(top_builddir)/kWnn/jutil
 
 INSTALL      = @INSTALL@
-INSTALLFLAGS = -o $(WNNOWNER)
+INSTALLFLAGS =
 
         ATOD = $(KWNNJUTILOBJ)/katod
         ATOF = $(KWNNJUTILOBJ)/katof
@@ -76,8 +76,7 @@
 install::
 	@case '${MFLAGS}' in *[i]*) set +e;; esac;
 	@for i in $(KWNNDICDIR); do if [ -d $(DESTDIR)$$i ]; then \
-	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i; \
-	chown $(WNNOWNER) $(DESTDIR)$$i); fi \
+	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i); fi \
 	done
 
 install:: $(ALLTGT)
