diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/Contrib/dic/gerodic/Makefile.in FreeWnn-1.1.1-a023/Contrib/dic/gerodic/Makefile.in
--- FreeWnn-1.1.1-a023.org/Contrib/dic/gerodic/Makefile.in	2006-05-28 18:37:02.000000000 +0200
+++ FreeWnn-1.1.1-a023/Contrib/dic/gerodic/Makefile.in	2023-02-02 20:00:31.846502039 +0100
@@ -46,7 +46,7 @@ top_builddir = @top_builddir@
 
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
-LOCAL_INSTFLAGS = -o ${WNNOWNER}
+LOCAL_INSTFLAGS =
 
 #ATOD=${JWNNBINDIR}/atod
 ATOD=${WNNJUTILOBJ}/atod
@@ -70,7 +70,6 @@ $(OBJ): $(SRC)
 install:: ${OBJ}
 	@if [ -d $(DESTDIR)$(GERODIR) ]; then set +x; \
 	else (set -x; $(MKDIRHIER) $(DESTDIR)$(GERODIR)); fi
-	chown ${WNNOWNER} $(DESTDIR)$(GERODIR)
 	$(INSTALL) $(INSTDATFLAGS) $(LOCAL_INSTFLAGS) ${OBJ} $(DESTDIR)$(GERODIR)
 	${WNNTOUCH} $(DESTDIR)$(GERODIR)/${OBJ}
 
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/Wnn/jserver/Makefile.in FreeWnn-1.1.1-a023/Wnn/jserver/Makefile.in
--- FreeWnn-1.1.1-a023.org/Wnn/jserver/Makefile.in	2010-07-11 12:18:17.000000000 +0200
+++ FreeWnn-1.1.1-a023/Wnn/jserver/Makefile.in	2023-02-02 20:00:31.846502039 +0100
@@ -36,7 +36,7 @@ include @top_builddir@/makerule.mk
 top_builddir = @top_builddir@
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
 
  JSERVERDEFINES = -DJSERVER -DWRITE_CHECK
        INCLUDES = -I$(WNNINCLUDESRC) -I$(top_builddir) -I$(srcdir)
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/Wnn/jutil/Makefile.in FreeWnn-1.1.1-a023/Wnn/jutil/Makefile.in
--- FreeWnn-1.1.1-a023.org/Wnn/jutil/Makefile.in	2011-04-17 18:29:50.000000000 +0200
+++ FreeWnn-1.1.1-a023/Wnn/jutil/Makefile.in	2023-02-02 20:00:31.846502039 +0100
@@ -37,7 +37,7 @@ top_builddir = @top_builddir@
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 LT_INSTALL      = $(LIBTOOL) --mode=install $(INSTALL)
 LT_LD           = $(LIBTOOL) --mode=link $(CCLINK)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
 
        INCLUDES = -I$(top_builddir) @EXTWNNJINC@ -I$(WNNINCLUDESRC)
         DEFINES = $(WNNDEFINES) $(WNNLANGDEF)
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/Wnn/pubdicplus/Makefile.in FreeWnn-1.1.1-a023/Wnn/pubdicplus/Makefile.in
--- FreeWnn-1.1.1-a023.org/Wnn/pubdicplus/Makefile.in	2013-11-22 11:53:01.000000000 +0100
+++ FreeWnn-1.1.1-a023/Wnn/pubdicplus/Makefile.in	2023-02-02 20:00:31.846502039 +0100
@@ -35,7 +35,7 @@ top_builddir = @top_builddir@
  WNNJUTILOBJ = $(top_builddir)/Wnn/jutil
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
 
        INCLUDES = -I$(top_builddir)
            ATOD = $(WNNJUTILOBJ)/atod
@@ -281,8 +281,7 @@ instserver:: install
 install::
 	@case '${MFLAGS}' in *[i]*) set +e;; esac;
 	@for i in $(JWNNDICDIR); do if [ -d $(DESTDIR)$$i ]; then \
-	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i; \
-	chown $(WNNOWNER) $(DESTDIR)$$i); fi \
+	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i); fi \
 	done
 
 install:: $(DIC)
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/Wnn/wnncons/dic/Makefile.in FreeWnn-1.1.1-a023/Wnn/wnncons/dic/Makefile.in
--- FreeWnn-1.1.1-a023.org/Wnn/wnncons/dic/Makefile.in	2001-08-14 15:43:21.000000000 +0200
+++ FreeWnn-1.1.1-a023/Wnn/wnncons/dic/Makefile.in	2023-02-02 20:00:31.846502039 +0100
@@ -33,7 +33,7 @@ include @top_srcdir@/makerule.mk
 top_builddir = ../../..
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
 
            ATOD = $(WNNJUTILSRC)/atod
        WNNTOUCH = $(WNNJUTILSRC)/wnntouch
@@ -74,8 +74,7 @@ instserver:: install
 install::
 	@case '${MFLAGS}' in *[i]*) set +e;; esac;
 	@for i in $(JWNNDICDIR); do if [ -d $(DESTDIR)$$i ]; then \
-	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i; \
-	chown $(WNNOWNER) $(DESTDIR)$$i); fi \
+	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i); fi \
 	done
 
 install:: $(DICS)
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/Xwnmo/xjutil/Makefile.in FreeWnn-1.1.1-a023/Xwnmo/xjutil/Makefile.in
--- FreeWnn-1.1.1-a023.org/Xwnmo/xjutil/Makefile.in	2001-08-14 15:43:21.000000000 +0200
+++ FreeWnn-1.1.1-a023/Xwnmo/xjutil/Makefile.in	2023-02-02 20:00:31.846502039 +0100
@@ -26,7 +26,7 @@ include @top_srcdir@/makerule.mk
 top_builddir = ../..
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
 
        INCLUDES = -I. $(XWNMOINCLUDES) -I$(XWNMOSRC)
 
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/Xwnmo/xwnmo/Makefile.in FreeWnn-1.1.1-a023/Xwnmo/xwnmo/Makefile.in
--- FreeWnn-1.1.1-a023.org/Xwnmo/xwnmo/Makefile.in	2001-08-14 15:43:21.000000000 +0200
+++ FreeWnn-1.1.1-a023/Xwnmo/xwnmo/Makefile.in	2023-02-02 20:00:31.846502039 +0100
@@ -26,7 +26,7 @@ include @top_srcdir@/makerule.mk
 top_builddir = ../..
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
 
        INCLUDES = $(XWNMOINCLUDES)
 
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/cWnn/cdic/Makefile.in FreeWnn-1.1.1-a023/cWnn/cdic/Makefile.in
--- FreeWnn-1.1.1-a023.org/cWnn/cdic/Makefile.in	2010-02-22 18:06:54.000000000 +0100
+++ FreeWnn-1.1.1-a023/cWnn/cdic/Makefile.in	2023-02-02 20:00:31.846502039 +0100
@@ -32,7 +32,7 @@ top_builddir = @top_builddir@
 CWNNJUTILOBJ = $(top_builddir)/cWnn/jutil
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
 
         ATOD = $(CWNNJUTILOBJ)/catod
         ATOF = $(CWNNJUTILOBJ)/catof
@@ -86,8 +86,7 @@ instserver:: install
 install::
 	@case '${MFLAGS}' in *[i]*) set +e;; esac;
 	@for i in $(CWNNDICDIR); do if [ -d $(DESTDIR)$$i ]; then \
-	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i; \
-	chown $(WNNOWNER) $(DESTDIR)$$i); fi \
+	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i); fi \
 	done
 
 install:: $(ALLTGT)
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/cWnn/jserver/Makefile.in FreeWnn-1.1.1-a023/cWnn/jserver/Makefile.in
--- FreeWnn-1.1.1-a023.org/cWnn/jserver/Makefile.in	2013-09-02 13:01:40.000000000 +0200
+++ FreeWnn-1.1.1-a023/cWnn/jserver/Makefile.in	2023-02-02 20:00:31.846502039 +0100
@@ -39,7 +39,7 @@ INSTALL      = @INSTALL@ $(INSTALLFLAGS)
  JSERVERDEFINES = -DJSERVER -DWRITE_CHECK
        INCLUDES = -I$(CWNNINCLUDESRC) -I$(top_builddir) -I$(srcdir)
         DEFINES = $(CWNNDEFINES) $(JSERVERDEFINES) $(WNNTERM) $(CWNNLANGDEF)
-LOCAL_INSTFLAGS = $(INSTUIDFLAGS) -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
   SERVERINITDEF = -DSERVER_INIT_FILE=\"/$(CWNNLANG)/cserverrc\"
     HINSIDEFINE = -DHINSIDATA_FILE=\"/$(CWNNLANG)/cixing.data\"
  TSERVERINITDEF = -DSERVER_INIT_FILE=\"/$(TWNNLANG)/tserverrc\"
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/cWnn/jutil/Makefile.in FreeWnn-1.1.1-a023/cWnn/jutil/Makefile.in
--- FreeWnn-1.1.1-a023.org/cWnn/jutil/Makefile.in	2011-04-17 18:29:50.000000000 +0200
+++ FreeWnn-1.1.1-a023/cWnn/jutil/Makefile.in	2023-02-02 20:00:31.846502039 +0100
@@ -36,7 +36,7 @@ top_builddir = @top_builddir@
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 LT_INSTALL      = $(LIBTOOL) --mode=install $(INSTALL)
 LT_LD           = $(LIBTOOL) --mode=link $(CCLINK)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
 
        INCLUDES = -I$(top_builddir) @EXTCWNNJINC@ -I$(CWNNINCLUDESRC)
         DEFINES = $(CWNNDEFINES) $(CWNNLANGDEF)
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/cWnn/tdic/Makefile.in FreeWnn-1.1.1-a023/cWnn/tdic/Makefile.in
--- FreeWnn-1.1.1-a023.org/cWnn/tdic/Makefile.in	2010-02-22 18:06:54.000000000 +0100
+++ FreeWnn-1.1.1-a023/cWnn/tdic/Makefile.in	2023-02-02 20:00:31.846502039 +0100
@@ -32,7 +32,7 @@ top_builddir = @top_builddir@
 CWNNJUTILOBJ = $(top_builddir)/cWnn/jutil
 
 INSTALL      = @INSTALL@
-INSTALLFLAGS = -o $(WNNOWNER)
+INSTALLFLAGS =
 
         ATOD = $(CWNNJUTILOBJ)/catod
         ATOF = $(CWNNJUTILOBJ)/catof
@@ -86,8 +86,7 @@ instserver:: install
 install::
 	@case '${MFLAGS}' in *[i]*) set +e;; esac;
 	@for i in $(TWNNDICDIR); do if [ -d $(DESTDIR)$$i ]; then \
-	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i; \
-	chown $(WNNOWNER) $(DESTDIR)$$i); fi \
+	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i); fi \
 	done
 
 install:: $(ALLTGT)
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/kWnn/jserver/Makefile.in FreeWnn-1.1.1-a023/kWnn/jserver/Makefile.in
--- FreeWnn-1.1.1-a023.org/kWnn/jserver/Makefile.in	2010-07-11 12:18:17.000000000 +0200
+++ FreeWnn-1.1.1-a023/kWnn/jserver/Makefile.in	2023-02-02 20:00:31.846502039 +0100
@@ -38,7 +38,7 @@ INSTALL      = @INSTALL@
  JSERVERDEFINES = -DJSERVER -DWRITE_CHECK
        INCLUDES = -I$(KWNNJSERVERSRC) -I$(KWNNINCLUDESRC) -I$(top_builddir)
         DEFINES = $(KWNNDEFINES) $(JSERVERDEFINES) $(WNNTERM) $(KWNNLANGDEF)
-LOCAL_INSTFLAGS = $(INSTUIDFLAGS) -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
   SERVERINITDEF = -DSERVER_INIT_FILE=\"/$(KWNNLANG)/kserverrc\"
     HINSIDEFINE = -DHINSIDATA_FILE=\"/$(KWNNLANG)/hinsi.data\"
         WRAPLIB = @WRAPLIB@
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/kWnn/jutil/Makefile.in FreeWnn-1.1.1-a023/kWnn/jutil/Makefile.in
--- FreeWnn-1.1.1-a023.org/kWnn/jutil/Makefile.in	2011-04-17 18:29:50.000000000 +0200
+++ FreeWnn-1.1.1-a023/kWnn/jutil/Makefile.in	2023-02-02 20:00:31.846502039 +0100
@@ -39,7 +39,7 @@ LT_LD           = $(LIBTOOL) --mode=link
 
        INCLUDES = -I$(top_builddir) @EXTKWNNJINC@ -I$(KWNNINCLUDESRC)
         DEFINES = $(KWNNDEFINES) $(KWNNLANGDEF)
-LOCAL_INSTFLAGS = -o $(WNNOWNER)
+LOCAL_INSTFLAGS =
     HINSIDEFINE = -DHINSIDATA_FILE=\"/$(KWNNLANG)/hinsi.data\"
 
 CLIENTPROGRAMS = kwddel kwdreg
diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/kWnn/kdic/Makefile.in FreeWnn-1.1.1-a023/kWnn/kdic/Makefile.in
--- FreeWnn-1.1.1-a023.org/kWnn/kdic/Makefile.in	2010-02-22 18:06:55.000000000 +0100
+++ FreeWnn-1.1.1-a023/kWnn/kdic/Makefile.in	2023-02-02 20:00:31.846502039 +0100
@@ -32,7 +32,7 @@ top_builddir = @top_builddir@
 KWNNJUTILOBJ = $(top_builddir)/kWnn/jutil
 
 INSTALL      = @INSTALL@
-INSTALLFLAGS = -o $(WNNOWNER)
+INSTALLFLAGS =
 
         ATOD = $(KWNNJUTILOBJ)/katod
         ATOF = $(KWNNJUTILOBJ)/katof
@@ -77,8 +77,7 @@ instserver:: install
 install::
 	@case '${MFLAGS}' in *[i]*) set +e;; esac;
 	@for i in $(KWNNDICDIR); do if [ -d $(DESTDIR)$$i ]; then \
-	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i; \
-	chown $(WNNOWNER) $(DESTDIR)$$i); fi \
+	set +x; else (set -x; $(MKDIRHIER) $(DESTDIR)$$i); fi \
 	done
 
 install:: $(ALLTGT)
