--- FreeWnn-1.1.1-a021/makerule.mk.in.orig	2012-11-17 14:43:37.699823983 +0100
+++ FreeWnn-1.1.1-a021/makerule.mk.in	2012-11-17 15:36:36.376424397 +0100
@@ -44,6 +44,8 @@
 
            MANDIR = @mandir@/man$(MANSUFFIX)
         LIBMANDIR = @mandir@/man$(LIBMANSUFFIX)
+          JMANDIR = @mandir@/ja/man$(MANSUFFIX)
+       JLIBMANDIR = @mandir@/ja/man$(LIBMANSUFFIX)
 
         MANSUFFIX = 1
      LIBMANSUFFIX = 3
--- FreeWnn-1.1.1-a021/Wnn/man/2.env/dic/Makefile.in.orig	2001-08-14 15:43:21.000000000 +0200
+++ FreeWnn-1.1.1-a021/Wnn/man/2.env/dic/Makefile.in	2012-11-17 15:40:28.603086222 +0100
@@ -34,6 +34,7 @@
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 
+MANDIR = $(JMANDIR)
 MANSUFFIX = 5
 
 all::
--- FreeWnn-1.1.1-a021/Wnn/man/2.env/wnn/Makefile.in.orig	2001-08-14 15:43:21.000000000 +0200
+++ FreeWnn-1.1.1-a021/Wnn/man/2.env/wnn/Makefile.in	2012-11-17 15:41:07.563085411 +0100
@@ -34,6 +34,7 @@
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 
+MANDIR = $(JMANDIR)
 MANSUFFIX = 4
 
 install.man:: 2a_ctrl.man
--- FreeWnn-1.1.1-a021/Wnn/man/3.libwnn/jl_lib/Makefile.in.orig	2001-09-16 16:20:17.000000000 +0200
+++ FreeWnn-1.1.1-a021/Wnn/man/3.libwnn/jl_lib/Makefile.in	2012-11-17 15:41:36.536418141 +0100
@@ -34,7 +34,7 @@
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 
-MANDIR = $(LIBMANDIR)
+MANDIR = $(JLIBMANDIR)
 MANSUFFIX = $(LIBMANSUFFIX)
 
 install.man:: bun_suu.man
--- FreeWnn-1.1.1-a021/Wnn/man/3.libwnn/js_lib/Makefile.in.orig	2001-08-14 15:43:21.000000000 +0200
+++ FreeWnn-1.1.1-a021/Wnn/man/3.libwnn/js_lib/Makefile.in	2012-11-17 15:41:55.896417735 +0100
@@ -34,7 +34,7 @@
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 
-MANDIR = $(LIBMANDIR)
+MANDIR = $(JLIBMANDIR)
 MANSUFFIX = $(LIBMANSUFFIX)
 
 all::
--- FreeWnn-1.1.1-a021/Wnn/man/3.libwnn/misc_lib/Makefile.in.orig	2001-08-14 15:43:21.000000000 +0200
+++ FreeWnn-1.1.1-a021/Wnn/man/3.libwnn/misc_lib/Makefile.in	2012-11-17 15:42:07.269750834 +0100
@@ -34,7 +34,7 @@
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 
-MANDIR = $(LIBMANDIR)
+MANDIR = $(JLIBMANDIR)
 MANSUFFIX = $(LIBMANSUFFIX)
 
 install.man:: msg_open.man
--- FreeWnn-1.1.1-a021/Wnn/man/3.libwnn/rk_lib/Makefile.in.orig	2001-08-14 15:43:21.000000000 +0200
+++ FreeWnn-1.1.1-a021/Wnn/man/3.libwnn/rk_lib/Makefile.in	2012-11-17 15:42:28.299750394 +0100
@@ -34,7 +34,7 @@
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 
-MANDIR = $(LIBMANDIR)
+MANDIR = $(JLIBMANDIR)
 MANSUFFIX = $(LIBMANSUFFIX)
 
 install.man:: rk_clear.man
--- FreeWnn-1.1.1-a021/Wnn/man/4.cmd/Makefile.in.orig	2001-08-14 15:43:21.000000000 +0200
+++ FreeWnn-1.1.1-a021/Wnn/man/4.cmd/Makefile.in	2012-11-17 15:42:54.219749853 +0100
@@ -34,6 +34,7 @@
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 
+MANDIR = $(JMANDIR)
 MANSUFFIX = 1
 
 install.man:: jserver.man
--- FreeWnn-1.1.1-a021/Wnn/man/6.jutil/Makefile.in.orig	2001-08-14 15:43:21.000000000 +0200
+++ FreeWnn-1.1.1-a021/Wnn/man/6.jutil/Makefile.in	2012-11-17 15:43:15.059749419 +0100
@@ -34,6 +34,7 @@
 
 INSTALL         = @INSTALL@ $(INSTALLFLAGS)
 
+MANDIR = $(JMANDIR)
 MANSUFFIX = 1
 
 install.man:: atod.man
