--- FreeWnn-1.1.1-a023/configure.in.orig	2016-09-27 19:05:57.673613326 +0200
+++ FreeWnn-1.1.1-a023/configure.in	2016-09-27 20:09:17.573569935 +0200
@@ -494,8 +494,8 @@
 dnl ### AC_CHECK_LIB(X11, main)
 
 AC_SEARCH_LIBS(crypt, crypt)
-AC_CHECK_LIB(socket, connect)
-AC_CHECK_LIB(bind, getservbyname)
+AC_SEARCH_LIBS(connect, socket)
+AC_SEARCH_LIBS(getservbyname, bind)
 AC_SEARCH_LIBS(gethostbyname, nsl)
 dnl libutil only needed by uum with openpty... (determine later)
 AC_CHECK_LIB(util, openpty, [HAVE_LIBUTIL='have_libutil'])
