--- FreeWnn-1.1.1-a021/Wnn/pubdicplus/Makefile.in.orig	2012-11-16 22:28:32.624376745 +0100
+++ FreeWnn-1.1.1-a021/Wnn/pubdicplus/Makefile.in	2012-11-17 13:19:15.953262821 +0100
@@ -421,7 +421,7 @@
 	$(LN) $? $@
 
 std.fzk: fzk.master-.c $(FZK_SRC)
-	$(CPP) $(FZK_FLAG) fzk.master-.c | egrep -v '^(# |#line |#pragma |$$)' | $(ATOF) -h $(HINSI) $@
+	$(CPP) -traditional-cpp $(FZK_FLAG) fzk.master-.c | egrep -v '^(# |#line |#pragma |$$)' | $(ATOF) -h $(HINSI) $@
 
 full.fzk: $(FZK_SRC)
 	$(MAKE) FZK_FLAG=-DKANSAI std.fzk
--- FreeWnn-1.1.1-a021/cWnn/cdic/Makefile.in.orig	2012-11-17 13:22:12.586592470 +0100
+++ FreeWnn-1.1.1-a021/cWnn/cdic/Makefile.in	2012-11-17 13:25:13.339922041 +0100
@@ -63,10 +63,10 @@
 $(DICS): $(HINSI)
 
 $(FULL):: $(FZK_SRCS) con.master-.c
-	$(CPP) -I$(srcdir) $(FZK_FLAG) con.master-.c | egrep -v '^(# |#line |#pragma |$$)' | $(ATOF) -h $(HINSI) $@
+	$(CPP) -traditional-cpp -I$(srcdir) $(FZK_FLAG) con.master-.c | egrep -v '^(# |#line |#pragma |$$)' | $(ATOF) -h $(HINSI) $@
 
 $(FULLR):: $(FZK_SRCS) con.masterR-.c
-	$(CPP) -I$(srcdir) $(FZK_FLAG) con.masterR-.c | egrep -v '^(# |#line |#pragma |$$)' | $(ATOF) -h $(HINSI) $@
+	$(CPP) -traditional-cpp -I$(srcdir) $(FZK_FLAG) con.masterR-.c | egrep -v '^(# |#line |#pragma |$$)' | $(ATOF) -h $(HINSI) $@
 
 # To avoid circular dependency, we don't use 'con.master?.c'
 con.master-.c: con.master
--- FreeWnn-1.1.1-a021/cWnn/tdic/Makefile.in.orig	2012-11-17 13:25:46.083254689 +0100
+++ FreeWnn-1.1.1-a021/cWnn/tdic/Makefile.in	2012-11-17 13:31:26.069914269 +0100
@@ -63,10 +63,10 @@
 $(DICS): $(HINSI)
 
 $(FULL):: $(FZK_SRCS) con.master-.c
-	$(CPP) -I$(srcdir) $(FZK_FLAG) con.master-.c | egrep -v '^(# |#line |#pragma |$$)' | $(ATOF) -h $(HINSI) $@
+	$(CPP) -traditional-cpp -I$(srcdir) $(FZK_FLAG) con.master-.c | egrep -v '^(# |#line |#pragma |$$)' | $(ATOF) -h $(HINSI) $@
 
 $(FULLR):: $(FZK_SRCS) con.masterR-.c
-	$(CPP) -I$(srcdir) $(FZK_FLAG) con.masterR-.c | egrep -v '^(# |#line |#pragma |$$)' | $(ATOF) -h $(HINSI) $@
+	$(CPP) -traditional-cpp -I$(srcdir) $(FZK_FLAG) con.masterR-.c | egrep -v '^(# |#line |#pragma |$$)' | $(ATOF) -h $(HINSI) $@
 
 # To avoid circular dependency, we don't use 'con.master?.c'
 con.master-.c: con.master
