--- FreeWnn-1.1.1-a017/Xsi/PubdicPlus/pod.c.orig	Sun Nov 12 16:34:27 2000
+++ FreeWnn-1.1.1-a017/Xsi/PubdicPlus/pod.c	Sun Nov 12 16:57:52 2000
@@ -25,6 +25,7 @@
 #endif
 
 #include <stdio.h>
+#include <stdlib.h>
 
 #define bzero(a, c) memset(a, 0, c)
 
--- FreeWnn-1.1.1-a017/Xsi/Wnn/jutil/atorev.c.orig	Sun Nov 12 17:14:23 2000
+++ FreeWnn-1.1.1-a017/Xsi/Wnn/jutil/atorev.c	Sun Nov 12 17:14:34 2000
@@ -35,6 +35,7 @@
 
 #include <stdio.h>
 #include <ctype.h>
+#include <stdlib.h>
 #include "commonhd.h"
 #include "jslib.h"
 #include "jh.h"
