diff -urNp -x '*.orig' FreeWnn-1.1.1-a023.org/Wnn/jd/jserverrc FreeWnn-1.1.1-a023/Wnn/jd/jserverrc
--- FreeWnn-1.1.1-a023.org/Wnn/jd/jserverrc	2001-06-14 20:08:31.000000000 +0200
+++ FreeWnn-1.1.1-a023/Wnn/jd/jserverrc	2023-02-02 20:00:50.813168705 +0100
@@ -43,6 +43,8 @@ readfile pubdic/tankan.dic
 readfile pubdic/bio.dic
 readfile gerodic/g-jinmei.dic
 
+readfile gerodic/g-jinmei.dic
+
 readfile pubdic/full.fzk
 
 ;readfile wnncons/tankan2.dic
